x-pack/test/security_solution_api_integration/test_suites/detections_response/detection_engine/rule_execution_logic/indicator_match/trial_license_complete_tier/indicator_match_alert_suppression.ts

```diff
@@ -1250,7 +1250,7 @@ export default ({ getService }: FtrProviderContext) => {
 
           await indexGeneratedSourceDocuments({
             docsCount: 60000,
-            interval: [firstTimestamp, '2020-10-28T05:35:50.000Z'],
+            interval: [firstTimestamp, '2020-10-28T05:45:50.000Z'],
```

Comment:
why interval boundaries needed to be changed?

Reply:
The problem was that the `firstTimestamp` was bigger than the end timestamp in that array:
```
 const firstTimestamp = '2020-10-28T05:45:00.000Z';
```
Since `firstTimstamp` was used more extensively I've considered changing the end part of the interval.

This "inverse" interval surfaced when swithcin the library to`@faker-js` which is more strict in the code logic and requires that the first parameter `from` be smaller than the `to` parameter.
The replaces library instead wasn't considering that logic and always created a date that wasn't between the interval, but was before that interval (`dateOffset` was negative and the `newDate` is created with a timestamp before the `firstTimestamp`).

from `faker` source (not anymore available on github, but available in our node_modules)
```
self.between = function (from, to) {
      var fromMilli = Date.parse(from);
      var dateOffset = faker.random.number(Date.parse(to) - fromMilli);

      var newDate = new Date(fromMilli + dateOffset);

      return newDate;
  };
...
```
This interval param was used in `x-pack/test/security_solution_api_integration/test_suites/detections_response/utils/data_generator/get_timestamp.ts`

Reply:
Happy to change the interval to any other value that you consider better, I don't see any error in the tests so I believe this change is safe.

Reply:
Thanks for catching and fixing this. Current configuration is fine.